recipes/default.rb

```diff
@@ -19,8 +19,10 @@
 
 influxdb_config node['influxdb']['config_file_path'] do
   config node['influxdb']['config']
+  notifies :restart, 'service[influxdb]', :delayed
 end
 
 service 'influxdb' do
-  action [:enable, :start]
+  supports(restart: true)
```

Comment:
hi @nilroy, the changes rubocop makes is all syntax reasons.  Function should be unaffected.

If the desire is to restart the service when there is a new `influxdb_config` change, then what you have will work.